scripts/zkai_native_d128_two_slice_verifier_execution_target_gate.py

```
@@ -0,0 +1,754 @@
+        str(OUTER_STATEMENT_ENVELOPE.relative_to(ROOT)),
+    ]
+    try:
+        completed = subprocess.run(
+            command,
+            cwd=ROOT,
+            check=True,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            timeout=120,
+        )
```

Ruff (0.15.12)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)